SU-SWS/stanford_profile
# D8CORE-4681 Enabled rabbit hole taxonomy support

# READY FOR REVIEW

# Summary
- Enabled rabbit hole module on taxonomy terms
- Allowed site builders and developers access to redirect terms (since it's not common, this access should suffice for now)

# Need Review By (Date)
- 8/12

# Urgency
- low

# Steps to Test
1. checkout this branch & `drush cim -y`
2. create a taxonomy term in the "News Topic" vocab
3. Set the rabbit hole setting to redirect to any other url, like 'http://google.com'
4. log out and try to view the taxonomy term page.
5. verify you are redirected to your url you configured.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
